R/carehomes.R

```diff
@@ -53,6 +53,11 @@ NULL
 ##'   value should be 1 (for the non-vaccinated group) and subsequent values be
 ##'   between 0 and 1.
 ##'
+##' @param waning_rate A single value or a vector of values representing the
```

Comment:
Are we ok with it only being immunity that wanes? (i.e., is this descriptive enough?)

Reply:
@edknock what do you think?

Reply:
I think it's fine.